src/app/ubs/ubs-admin/components/add-violations/add-violations.component.ts

```diff
@@ -23,28 +22,28 @@ interface DataToSend {
   imagesToDelete?: string[] | null;
 }
 
+interface ViolationImage {
+  src: string;
+  name: string | null;
+  file: File;
+}
+
```

Comment:
bad practice, interfaces should be move out of the component

Reply:
It is not needed anywhere outside, should I remove it completely?